src/components/menu-bar/menu-bar.jsx

```diff
@@ -26,7 +26,9 @@ import {
     getIsUpdating,
     getIsShowingProject,
     requestNewProject,
-    saveProject
+    remixProject,
+    updateProject,
```

Comment:
Why `update` instead of `save`?

Reply:
I think "save" is ambiguous. Remixing involved saving. Saving as a copy involves saving. Creating involves saving. "Update" tells you that we have a project on the server already, it has an id, and we are just updating that. I think it will be clearer.